services/victoriametrics/victoriametrics.go

```diff
@@ -147,13 +147,6 @@ func (svc *Service) updateConfiguration(ctx context.Context) error {
 func (svc *Service) RequestConfigurationUpdate() {
 	select {
 	case svc.sema <- struct{}{}:
-		ctx, cancel := context.WithTimeout(context.Background(), configurationUpdateTimeout)
-		defer cancel()
-		err := svc.updateConfiguration(ctx)
-		if err != nil {
-			svc.l.WithError(err).Errorf("cannot reload configuration")
-		}
-
```

Comment:
Why?

Reply:
I'm not fully uderstood idea of that code, but it reloads configuration twice. First time here, and the second on other side of `sema` channel. Also there is some kind of protecion of too often config updation, bit it doesn't work until `sema` channel is buffered.

Probably I just missed the idea

Reply:
> but it reloads configuration twice

Indeed, there seems to be a mistake.

Please revert this and related changes in this PR, I will take a look in a separate branch

Reply:
Ok, will cherry pick them to another branch

Reply:
https://github.com/percona/pmm-managed/pull/595

Reply:
Reverted